snap/snapcraft.yaml

```diff
@@ -11,46 +11,64 @@ confinement: classic
 
 platforms:
   amd64:
-  # arm64:
+  arm64:
 
 apps:
   yazi:
     command: yazi
     environment:
-      PATH: $SNAP/bin:$PATH
+      PATH: $SNAP/bin:$SNAP/usr/bin:$PATH
 
 parts:
   yazi:
     plugin: rust
     source: https://github.com/sxyazi/yazi.git
-    build-packages:
-      - wget
+    stage-snaps:
+      - jq
+    stage-packages:
+      - 7zip-standalone
+      - chafa
+      - fd-find
+      - ffmpeg
+      - fzf
+      - libpoppler-cpp0t64
```

Comment:
The name `libpoppler-cpp0t64` sounds like a dynamic library - does it provide the `pdftoppm` binary?

Reply:
Oops, good catch. I've swapped this for `poppler-utils`, which will indeed pull in this library, but also provide the `pdftoppm` binary explicitly.